### PR TITLE
Desktop version of portfolio website

### PR DESCRIPTION
**Pull request by zmekonnen251**
## In this project I:
- Applied styling for the desktop version.
- Used media queries to make the website responsive.
- Implemented user experience by using the hovering effect.
- Wrote a descriptive pr.
- Wrote a descriptive readme.
- Applied basic styling by considering both the mobile and desktop versions.

Thanks for the review!!! :smile:

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -6,11 +6,20 @@
...
+/* Headline styling  */
+
 .header {
   background-color: #3c3a39;
   color: #fff;
   padding: 10px 24px;
-  box-sizing: border-box;
+}
+
+@media (min-width: 920px) {
```
Comment (Meltrust) [inline style.css]: - [x] Your `min-width` media queries are out of the required range.  Let's kindly them it to any range from 992px to 1100px.

image

That way, we will meet the project requirements, and the desktop view will be more consistent.

File: style.css [hunk 2]
```diff
@@ -39,6 +48,29 @@
   margin-bottom: 120px;
 }
 
+@media (min-width: 920px) {
```
Comment (Meltrust) [inline style.css]: - [x] Your `min-width` media queries are out of the required range.  Let's kindly them it to any range from 992px to 1100px.

image

That way, we will meet the project requirements, and the desktop view will be more consistent.- [ ] Your `min-width` media queries are out of the required range.  Let's kindly them it to any range from 992px to 1100px.

File: style.css [hunk 3]
```diff
@@ -39,6 +48,29 @@
...
+  }
+}
+
+.headline-text {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+@media (min-width: 920px) {
```
Comment (Meltrust) [inline style.css]: - [x] Your `min-width` media queries are out of the required range.  Let's kindly them it to any range from 992px to 1100px.

image

That way, we will meet the project requirements, and the desktop view will be more consistent.

File: style.css [hunk 4]
```diff
@@ -50,6 +82,13 @@
   padding: 0 24px;
 }
 
+@media (min-width: 920px) {
```
Comment (Meltrust) [inline style.css]: - [x] Your `min-width` media queries are out of the required range.  Let's kindly them it to any range from 992px to 1100px.

image

That way, we will meet the project requirements, and the desktop view will be more consistent.

File: style.css [hunk 5]
```diff
@@ -70,27 +109,59 @@
   color: #fff;
 }
 
+@media (min-width: 920px) {
```
Comment (Meltrust) [inline style.css]: - [x] Your `min-width` media queries are out of the required range.  Let's kindly them it to any range from 992px to 1100px.

image

That way, we will meet the project requirements, and the desktop view will be more consistent.

File: style.css [hunk 6]
```diff
@@ -70,27 +109,59 @@
...
 .social-media {
   width: 200px;
   height: 40px;
   margin-left: 24px;
   margin-bottom: 130px;
 }
 
+/* social media links for desktop version */
+
+@media (min-width: 920px) {
```
Comment (Meltrust) [inline style.css]: - [x] Your `min-width` media queries are out of the required range.  Let's kindly them it to any range from 992px to 1100px.

image

That way, we will meet the project requirements, and the desktop view will be more consistent.

File: style.css [hunk 7]
```diff
@@ -70,27 +109,59 @@
...
+}
+
 .social-media-links {
   display: flex;
   list-style: none;
   gap: 15px;
   padding: 9px 9px;
 }
 
+@media (min-width: 920px) {
```
Comment (Meltrust) [inline style.css]: - [x] Your `min-width` media queries are out of the required range.  Let's kindly them it to any range from 992px to 1100px.

image

That way, we will meet the project requirements, and the desktop view will be more consistent.

File: style.css [hunk 8]
```diff
@@ -70,27 +109,59 @@
...
+}
+
 .illustration-header-mobile {
   position: absolute;
   right: 0%;
   bottom: -8%;
   z-index: 1;
 }
 
+@media (max-width: 920px) {
```
Comment (Meltrust) [inline style.css]: - [x] Your `min-width` media queries are out of the required range.  Let's kindly them it to any range from 992px to 1100px.

image

That way, we will meet the project requirements, and the desktop view will be more consistent.

File: style.css [hunk 9]
```diff
@@ -326,19 +391,32 @@
...
 
-.about_me_illustration {
+.about_me_illustration_mobile {
   position: absolute;
+  bottom: -180px;
   right: 0;
-  bottom: 150px;
+}
+
+@media (max-width: 920px) {
```
Comment (Meltrust) [inline style.css]: - [x] Your `min-width` media queries are out of the required range.  Let's kindly them it to any range from 992px to 1100px.

image

That way, we will meet the project requirements, and the desktop view will be more consistent.- [ ] Your `min-width` media queries are out of the required range.  Let's kindly them it to any range from 992px to 1100px.

File: style.css [hunk 10]
```diff
@@ -432,11 +518,23 @@ button:hover {
...
   border: none;
   padding: 12px;
   width: 327px;
+  border-bottom: 1px solid #000;
+  font-family: "Roboto", Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+@media (max-width: 920px) {
```
Comment (Meltrust) [inline style.css]: - [x] Your `min-width` media queries are out of the required range.  Let's kindly them it to any range from 992px to 1100px.

image

That way, we will meet the project requirements, and the desktop view will be more consistent.

File: style.css [hunk 11]
```diff
@@ -464,20 +579,254 @@ button:hover {
...
-
 .social-media-footer {
-  margin-top: 25px;
-  margin-bottom: 36px;
-  margin-left: 0;
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+@media (min-width: 920px) {
```
Comment (Meltrust) [inline style.css]: - [x] Your `min-width` media queries are out of the required range.  Let's kindly them it to any range from 992px to 1100px.

image

That way, we will meet the project requirements, and the desktop view will be more consistent.

File: index.html [hunk 12]
```diff
@@ -51,37 +65,44 @@ <h1 class="main-header">Hey There. I'm Zelalem</h1>
...
+        class="illustration-header-desktop"
+      />
     </article>
 
-    <h2 class="works_heading">My Recent Works</h2>
-
-    <hr class="horizontal_line" />
-
+    <div class="work_heading_block" id="portfolio">
+      <h2 class="works_heading">My Recent Works</h2>
```
Comment (Meltrust) [inline index.html]: - [ ] This `<h2>` should span just one line according to the template.  Let's kindly correct it.
- [ ] In these headings for each project, the descriptions, and the icons, there are many either omitted or wrong rules (for the desktop view) such as font-size, line-height, etc.  Please kindly correct them.
- [ ] There is a gap here that shouldn't be there:

image

Let's please kindly correct it so the left main container takes 60% of the page, and the right container the other 40%.

That way, your project will be more consistent and will look a lot like the required template.

File: index.html [hunk 13]
```diff
@@ -97,7 +118,7 @@ <h3 class="project_heading">Professional Art Printing Data</h3>
         <button class="see_project button">See Project</button>
       </div>
 
-      <section class="work_card">
+      <section class="work_card card">
```
Comment (Meltrust) [inline index.html:137]: - [ ] Kindly implement a `::hover` pseudoclass here, and over all those six projectsso when the user hovers the element, the opacity is gone and a button appears to go to the project.  You could use `display-none` for the language tags, and a `display-block` for the button on hover.  That way, your hovers on the desktop will look beautiful and functional 💪

File: index.html [hunk 14]
```diff
@@ -234,52 +261,68 @@ <h1 class="about_myself_heading">About me</h1>
...
+          />
+
+          <input
+            class="last_name"
+            type="text"
+            placeholder="Last Name"
+            name="last_name"
+            maxlength="30"
+            required
+          />
```
Comment (Meltrust) [inline index.html]: - [ ] Kindly please be mindful of the project requirements.  Let's make these form fields just one that contains the whole name of the person.

image

File: index.html [hunk 15]
```diff
@@ -178,27 +199,33 @@ <h3 class="project_heading">Professional Art Printing Data</h3>
...
-        <p class="about_me_description">
-          Hello I'm a software developer! I can help you build a product ,
-          feature or website Look through some of my work and experience! If you
-          like what you see and have a project you need coded, don’t hestiate to
-          contact me.
-        </p>
-
-        <button class="get_my_resume button">Get My Resume</button>
+        <div class="aboutme_description">
+          <h1 class="about_myself_heading">About me</h1>
```
Comment (Meltrust) [inline index.html:231]: - [ ] This "about me" headline should be a little larger (width-wise).   Kindly let's fix it so the left element takes some more of the page.

image

File: index.html (hunk 13 above)
Comment (Meltrust) [inline index.html:137]: - [ ] Kindly implement a `::hover` pseudoclass here, so when the user hovers the element, the opacity is gone and a button appears to go to the project.  You could use `display-none` for the language tags, and a `display-block` for the button on hover.  That way, your hovers on the desktop will look beautiful and functional 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @zmekonnen251, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please mind your CSS properties, a lot of them are different from the mobile in the figma.  Be specially mindful of line-hight, font-size, etc

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 14 above)
Comment (zmekonnen251) [reply index.html]: I was also confused about this part of the design, the input fields are different in the mobile and desktop versions. In the mobile version, there is the only full name, email, and text area. But in the desktop version of the UI, there is First name, Last name, Email, and text area. I was just trying to make it the template.

**Comment by zmekonnen251** [comment]
I fixed all the comments @Meltrust  gave me.

File: index.html [hunk 16]
```diff
@@ -51,37 +65,51 @@ <h1 class="main-header">Hey There. I'm Zelalem</h1>
...
+            <li>Ruby</li>
+          </ul>
 
-        <button class="see_project_multi_post button">See Project</button>
+          <button class="see_project_multi_post button">See Project</button>
+        </div>
       </section>
 
-      <div class="work_card">
+      <div class="work_card first_card">
```
Comment (Meltrust) [inline index.html]: - [ ] You implemented the hover button well, but not the opacity.  Kindly now try to do the following:
When the user hovers ANY part of the container with the mouse:
- The button appears.
- The title of the project, the descriptions, and the language tags disappear (you could use display: none or visibility: hidden).
- The image of the project is totally visible, along with the "See project" button.

image

This is how each project should look on ::hover:

image

Look:

image

In this part of the Figma (which is confusing), the first project looks like this just to tell you how each of them should look when hovered by the mouse. 

To recap:

Default state (NOT HOVERED) | Hovered state
:---:|:---:
image |  image 

Kindly please implement the ::hover for all the 6 project containers.  Your project will look impressive! 💪

File: index.html [hunk 17]
```diff
@@ -234,52 +268,50 @@ <h1 class="about_myself_heading">About me</h1>
...
-        <button class="submit_button" type="submit">Get in touch</button>
-      </form>
-    </section>
-
-    <hr class="horizontal_line horizontal_line_3" />
-
     <footer class="footer">
-      <div class="social-media social-media-footer">
-        <ul class="social-media-links">
+      <section class="contact_section" id="contact">
```
Comment (Meltrust) [inline index.html:291]: - [ ] Good job correcting some of the fonts! 💯 , however, all over your project (mostly in the about me section) there are still discrepancies with the font styles shown on the Figma design.  

Look: 

image

As shown above, you just need to select the text in question, and look at the font properties on the right of the template.

Put special attention on the line-height, font-size, font-spacing, font-weight, font-family, letter-spacing.

Let's please kindly correct them so your project looks just like the Figma 💯

File: index.html [hunk 18]
```diff
@@ -234,52 +268,50 @@ <h1 class="about_myself_heading">About me</h1>
...
+        <h3>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+
+        <form
```
Comment (Meltrust) [inline index.html:297]: - [ ] Good job making the name of the user a single field!  Now let's make the form fields a little bigger horizontally to make them look like the figma:

image

The figma:

image

That way, your form will be hugely more consistent! 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @zmekonnen251, 

Great work on making the changes requested by the previous review 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zmekonnen251** [comment]
I fixed all the changes required me to do.

Thanks for the review!!! :smile:

File: index.html [hunk 19]
```diff
@@ -234,52 +287,50 @@ <h1 class="about_myself_heading">About me</h1>
...
+
+          <textarea
+            class="message"
+            name="message"
+            id="message"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            placeholder="Enter text here"
+            required
```
Comment (Meltrust) [inline index.html:326]: - [ ] OPTIONAL: I can see you did a good job there trying to achieve the requirement... But you went too far and this even introduced a horizontal scroll overflow! 

image


Just please kindly I suggest you correct it.  (I can see you implemented a `width: 150%` on each element.  I tested reducing it to 130% and it worked wonderfully!!!). 😄

**Review by Meltrust — APPROVED** [review]
Hi @zmekonnen251, 

## I can see you worked really hard implementing all the required changes, good job!!! Just please read and try to implement the optional comment I made below. 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 19 above)
Comment (zmekonnen251) [reply index.html:326]: Thank you very much for helping me to make my project looks similar to the template. 👍

File: index.html [hunk 20]
```diff
@@ -51,154 +72,186 @@ <h1 class="main-header">Hey There. I'm Zelalem</h1>
         alt=""
         class="illustration-header-mobile"
       />
+      <img
+        src="img/Illustration-header-desktop.png"
+        alt="healine illustration"
+        class="illustration-header-desktop"
+      />
```
Comment (souad988) [inline index.html:79]: i suggest you use background image rather than img tag, i think it is less code in the html file and you don't need to add css styling for the img

**Review by souad988 — COMMENTED** [review]
Hi @zmekonnen251,

Good job so far!
There are some little changes that i suggest you do

Required Changes ♻️
Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
